code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm

```diff
@@ -333,13 +333,39 @@
 			M.AdjustWeakened(2)
 	..()
 
+	if(reac_volume > 11)
+		metabolization_rate = 1 * REAGENTS_METABOLISM
```

Comment:
1) This should be on touch only since it's basically getting the splash code to make people orange just like if they'd overdosed, and every other method can overdose.
2) The metabolization rate here is uneccesary, splashing does not cause the reagents to be ingested.
3) The two if(prob)s should go, they're meant to be checked every tick overdose_process is called so they happen every so often during an overdose, not just once.